.github/workflows/dockerhub.yml

```diff
@@ -6,7 +6,6 @@ on:
       - "carma-develop"
       - "master"
       - "release/*"
-      - "release/lavida"
 jobs:
   dockerhub:
     runs-on: ubuntu-latest-8-cores
```

Comment:
Line 28 needs to change, it has candidate info:
`tags: usdotfhwastolcandidate/${{ github.event.repository.name }}:lavida`

Reply:
That line is just plain incorrect. I replaced it with dockerhub stol action. This is mostly duplication from our dockerhub automated builds. It looks like for autoware.ai we use dockerhub automated builds for everything. GitHub actions is faster than dockerhub automated builds. I can either do these changes or remove the file all together. We are eventually going to move from dockerhub automated builds to Github action so I figured the current change is best.